### PR TITLE
implemented periodic change o reference speed

### PR DESCRIPTION
**Pull request by matthewtuby**

### CODE REVIEW

**Review by aabbeell — APPROVED** [review]
ok

File: train-user/src/main/java/hu/bme/mit/train/user/TrainUserImpl.java [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 	private TrainController controller;
 	private int joystickPosition;
+	private Thread thr;
```
Comment (aabbeell) [inline train-user/src/main/java/hu/bme/mit/train/user/TrainUserImpl.java:10]: very nice sir!

**Review by aabbeell — APPROVED** [review]
very nice code, sir

File: train-user/src/main/java/hu/bme/mit/train/user/TrainUserImpl.java (hunk 1 above)
Comment (matthewtuby) [reply train-user/src/main/java/hu/bme/mit/train/user/TrainUserImpl.java:10]: thanks

File: train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java [hunk 2]
```diff
@@ -2,11 +2,15 @@
 
 import hu.bme.mit.train.interfaces.TrainController;
 
-public class TrainControllerImpl implements TrainController {
+import java.util.Timer;
+import java.util.TimerTask;
+
```
Comment (aabbeell) [inline train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java]: delete whitespace

**Review by aabbeell — CHANGES_REQUESTED** [review]

**Review by aabbeell — APPROVED** [review]
ok

**Pull request merged**